sovtoken/sovtoken/client_authnr.py

```diff
@@ -53,10 +53,16 @@ def authenticate(self, req_data, identifier: str = None,
     # The verkey will be unobtainable from input, it will raise CouldNotAuthenticate
     # Raises UnknownIdentifier if input is not a valid base58 value
     def authenticate_xfer(self, req_data, verifier):
+        extra_fields = []
+        if req_data[OPERATION].get(EXTRA, None):
+            extra_fields.append(req_data[OPERATION][EXTRA])
+        if req_data.get(f.TAA_ACCEPTANCE.nm, None):
```

Comment:
As for TAA, we expect it in `req_data` (as for any other request). Putting EXTRA here is done just to resolve TODO comment in `verify_signtures_on_payments`, and we expect it always empty on practice, right?

Reply:
We are putting it here to have a signature verification mechanism similar to SDK. They returned extra to the request and signature, we should do it as well.